cfpb/consumerfinance.gov
# Remove `content-l__main` class

The corresponding removal of this class in the DS is happening in https://github.com/cfpb/design-system/pull/1929

## Removals

- Remove `content-l__main` class

## How to test this PR

1. `yarn build` and visit the newsroom. Check the pre-footer and resize the page. With this change the stacking of the items in the pre-footer changes orientation in the tablet range for the page.


## Screenshots

Before:
<img width="714" alt="Screenshot 2024-03-14 at 2 08 45 PM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/a77a0481-a25d-4da1-937b-ad584ab8e352">

After:
<img width="697" alt="Screenshot 2024-03-14 at 2 08 54 PM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/34dcd1be-9d4e-4b0d-b473-69130d8cca1f">
